theme/templates/resource-landing-page/modals.html

```diff
@@ -524,13 +530,75 @@ <h4 class="modal-title" id="copy-resource-title">Copy This Resource</h4>
+                        <span>{{ rights.statement|linebreaks }}</span>
+                    </div>
+                {% else %}
+                    <span>{{ rights.statement|linebreaks }}</span>
+                {% endif %}
+                <span class="rights-url"><a href="{{ rights.url }}">&nbsp{{ rights.url }}</a></span>
+                <p style="margin-top: 5px">
+                In downloading this resource contents you are ethically bound to respect the terms of this license.
+                Please confirm that you accept the terms of this license below before you can do any downloads for this resource.
+                </p>
+                <input type="checkbox" id="agree-chk-download-bag"><label for="agree-chk-download" class="checkbox-label">I accept the terms of this license.</label>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                <a type="button" class="btn btn-success" id="download-bag-btn"
+                   href="{{ cm.bag_url }}" disabled>Download</a>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade" id="license-agree-dialog-file" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                <h4 class="modal-title" id="license-agree-title">Resource License Agreement</h4>
+            </div>
+            <div class="modal-body">
+                {% if rights.statement|length > 1000 %}
+                    <div class="pre-scrollable">
+                        <span>{{ rights.statement|linebreaks }}</span>
+                    </div>
+                {% else %}
+                    <span>{{ rights.statement|linebreaks }}</span>
+                {% endif %}
+                <span class="rights-url"><a href="{{ rights.url }}">&nbsp{{ rights.url }}</a></span>
+                <p style="margin-top: 5px">
+                In downloading this resource contents you are ethically bound to respect the terms of this license.
+                Please confirm that you accept the terms of this license below before you can do any downloads for this resource.
+                </p>
+                <input type="checkbox" id="agree-chk-download-file"><label for="agree-chk-download-file" class="checkbox-label">I accept the terms of this license.</label>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                <button type="button" class="btn btn-success" id="download-file-btn"
+                   disabled>Download</button>
+            </div>
+        </div>
+    </div>
+</div>
+
```

Comment:
@hyi I tried to use one dialog but I am having trouble in displaying the message for bag creation in browser firefox and IE that uses an iframe for the bag download to occur. Have spent quite a bit of time on this. I am leaving this as it is - two separate dialogs.